### DISCUSSION TITLE
关于web版本的ide对于文件本地读写的问题

### DISCUSSION THREAD

**Discussion by andysama-work** [opensumi/core#3209] [Q&A]
我使用web版本部署在线上的话只能打开服务器的文件资源请问如何修改file-service达到想要的效果（类似于vscode.dev那种效果
）

**Comment by hacke2** [comment]
https://github.com/opensumi/codeblitz 集成这个项目试一下，你说的文件系统已经实现了，可以访问 https://codeblitz.cloud.alipay.com/gitee/opensumi/codeblitz 体验一下

**Reply by andysama-work** [reply]
你这个是打开gethub仓库的文件?

**Reply by andysama-work** [reply]
web版本的ide如何自定义file-serice呢

**Reply by andysama-work** [reply]
OpenSumi 也支持通过自定义 file-service 的方式来实现其他文件系统，例如 MemoryFS、BrowserFS 等。就是你们文档上的这句话

**Reply by hacke2** [reply]
参考下 https://codeblitz.cloud.alipay.com/zh/docs/develop/file-system/ 看下 CodeBlitz 文件系统是否能满足

**Reply by andysama-work** [reply]
我想使用browser-fs-access去操作文件不是通过uri、路径啥的操作

**Comment by sjzpengyuyan** [comment]
有进展吗？我也遇到了同样的问题
